README.md

```diff
@@ -76,7 +76,7 @@ console.log(await driver.getOrientation()); // -> 'LANDSCAPE'
 | `getCookies`                      | 
 | `getCssProperty`                  | 
 | `getCurrentContext`               |
-| `getDeviceDateAndTime`            | 
+| `getDeviceTime`       		    | 
```

Comment:
Spacing looks fine in my sublime.Not sure what's going wrong here!. Will check and update again